bugbug/model.py

```diff
@@ -284,9 +298,19 @@ def train(self, importance_cutoff=0.15):
                 f"\nConfidence threshold > {confidence_threshold} - {len(y_test_filter)} classified"
             )
             if len(y_test_filter) != 0:
+                confusion_matrix = metrics.confusion_matrix(
```

Comment:
Since you are doing the same thing twice, it might be better to define a helper function which takes in input y_test, y_pred and class_names and returns the output to print.

Reply:
E.g. `def get_labeled_confusion_matrix(y_test, y_pred, class_names):`